### PR TITLE
Mute edited_ip_detect_script_yields_new_packages on Windows

### PR DESCRIPTION
**Pull request by sschneid**
### High-level description
`edited_ip_detect_script_yields_new_packages` was written before having Windows unit testing was enabled on PRs, and is thusly failing.

### Corresponding DC/OS tickets
These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

[DCOS_OSS-3456](https://jira.mesosphere.com/browse/DCOS_OSS-3456)- edited_ip_detect_script_yields_new_packages has muted Windows tests requiring investigation

### CODE REVIEW

**Comment by sschneid** [comment]
https://teamcity.mesosphere.io/viewLog.html?buildId=1046360

@mesosphere-mergebot label Ready For Review

**Comment by sschneid** [comment]
@mesosphere-mergebot label Ready For Review

File: gen/tests/test_config.py [hunk 1]
```diff
@@ -815,6 +815,7 @@ def test_exhibitor_admin_password_obscured():
     assert yaml.load(generated.arguments['config_yaml_full'])[var_name] == var_value
 
 
+@pytest.mark.skipif(pkgpanda.util.is_windows, reason='TODO: Needs porting on Windows')
```
Comment (orsenthil) [inline gen/tests/test_config.py]: Hi @sschneid  - Can we add a JIRA against TODO. We have a recommendation on how to add TODO comment here - https://github.com/dcos/dcos/blob/master/docs/styleguides/python.md

**Comment by mesosphere-mergebot2** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by sschneid** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
Your pull request's branch is not based on the most recent version of `master`. Please rebase your changes against this repo's `master` branch.

**Comment by sschneid** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by sschneid** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ship-it

**Comment by orsenthil** [comment]
@mesosphere-mergebot merge-it

**Pull request merged**